distr/flecs.h

```diff
@@ -15659,14 +15659,19 @@ typedef struct ecs_enum_constant_t {
     const char *name;
 
     /** May be set when used with ecs_enum_desc_t */
-    int32_t value;
+    int64_t value;
+
+    /** For when the underlying type is unsigned */
+    uint64_t value_unsigned;
```

Comment:
Why not a union?

Reply:
This type is used in `ecs_enum_desc_t`/`ecs_bitmask_desc_t` so it needs to be easy to assign. If it was turned into a union, you would have to set two fields, one to indicate which branch is used, and one to set the actual value.